packages/oui-navbar/README.md

```diff
@@ -149,9 +202,10 @@ Define the base styles of a backdrop for a navbar.
 
 ```less
 #oui > .navbar-backdrop(
```

Comment:
There is no example with navbar-backdrop

Reply:
This need javascript to toggle value of `aria-expanded` to be visible. 
Useless if put in the examples without context.

We'll need a single page with a full example for this (like for an example for fixed menu).